### PR TITLE
add jenkins + terraform

### PR DESCRIPTION
**Pull request by rodriguezsergio**

### CODE REVIEW

File: terraform/stubbornjava.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    default = {}
+}
+
+# TODO: import stubbornjava-webapp
+
+resource "aws_instance" "ci" {
+  count                       = 1
+  ami                         = "${var.amis["amazon-linux-2017-09"]}"
+  disable_api_termination     = true
+  instance_type               = "t2.micro"
```
Comment (billoneil) [inline terraform/stubbornjava.tf]: image

Comment (rodriguezsergio) [reply terraform/stubbornjava.tf]: :D

File: ansible/install_roles.yml [hunk 2]
```diff
@@ -1,4 +1,13 @@
-# ansible-galaxy install -r install_roles.yml
+# ansible-galaxy install --roles-path=galaxy_roles/ -r install_roles.yml 
 
 - src: geerlingguy.java
   version: 1.7.4
+
+- src: geerlingguy.jenkins
+  version: 3.2.1
+
+- src: savoirfairelinux.nexus3-oss
```
Comment (rodriguezsergio) [inline ansible/install_roles.yml]: Just realized this was old copy/pasta. I'll remove this.

**Comment by billoneil** [comment]
Looks pretty good to me. I'd like to split it into modules at some point so I could do something like this if possible.

prod.tf
env = prod
instanceType = t2.small
servers = 4

staging.tf
env = staging
instaneType = t2.micro
servers = 1

**Comment by billoneil** [comment]
Going to merge for now and we can make modules / lock down jenkins later

**Comment by rodriguezsergio** [comment]
@billoneil yeah we can make modules later. It shouldn't be too difficult.

**Pull request merged**